config/install.yaml

```diff
@@ -419,7 +428,7 @@ data:
     syncTimeIntervalMs: 30000
     autoHealEnabled: true
     includedResources: "group=apps,kind=Deployment;\
-    group=,kind=ConfigMap;group=,kind=Secret;group=,kind=ServiceAccount;group=,kind=Namespace;\
```

Comment:
so, these are the Resources that gitops-engine keeps track of that we're syncing, right? I think we only need `Secret` for ourselves to be able to read in Git credentials, so it doesn't need to be here, does it?

Reply:
I guess we can keep it for open source version in case pipeline needs secret as well

Reply:
If we include `Secret`, then maybe we should also include `Role` and `RoleBinding` in case their Pipeline Pods' Service Accounts need to be bound to Roles.

Reply:
Also, the verbs for the role need to be updated to “*” if we include Secrets and Namespaces here, right?

also, what about Service kind?

Reply:
yeah, in that case, we should allow write permission for `Secret`. so let me just take out `Secret` as it really doesn't matter, the user can patch on it.